Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/opendatahub/text-generation-inference:stable-406b404
+FROM quay.io/opendatahub/text-generation-inference:fast-66ff883
```

Comment:
What is the reason to change from stable to fast? Is it just a rename (aka stable is not there anymore)?

Reply:
Do we also need to update the tag in `docs/caikit-servingruntime.yaml` ?

Reply:
So, that's there because I need to update CI to be able to build the identical stable tag

fast-66ff883 should be identical to the stable

As for updating, no, shouldn't need to update it there, since this PR will be the new image to update there